src/openzaak/templates/admin/change_list.html

```diff
@@ -0,0 +1,24 @@
+{% extends "admin/change_list.html" %}
+
+{% load i18n admin_urls %}
+
+{% block object-tools-items %}
+  {% comment %} {% if has_add_permission %}
+  {<li>
+    {% url cl.opts|admin_urlname:'add' as add_url %}
+    <a href="{% add_preserved_filters add_url is_popup to_field %}" class="addlink">
+      {% blocktrans with cl.opts.verbose_name as name %}Add {{ name }}{% endblocktrans %}
+    </a>
+  </li
+  {% endif %} {% endcomment %}
+  {{ block.super }}
+    {% if cl.opts.verbose_name == "catalogus" %}
```

Comment:
Since this code is only for the catalogus admin, I prefer making a new (copy) template file and point to it from the admin class (something like `template_name=...`) instead of using an if-statement.